src/Shell.php

```diff
@@ -1609,7 +1609,7 @@ protected function writeVersionInfo()
         try {
             $client = $this->config->getChecker();
             if (!$client->isLatest()) {
-                $this->output->writeln(\sprintf('<whisper>New version is available (current: %s, latest: %s)</whisper>', self::VERSION, $client->getLatest()));
+                $this->output->writeln(\sprintf('<whisper>New version is available @ psysh.org/psysh (current: %s, latest: %s)</whisper>', self::VERSION, $client->getLatest()));
```

Comment:
let's spell out `at` rather than using `@` (since it's only one more character :P)